### ISSUE TITLE
Trick LG AC to lower temperatures

### ISSUE THREAD

**Issue by rrooggiieerr** [JanM321/esphome-lg-controller#90]
I'm using my LG AC to heat my apartment. The lowest temperature my LG AC allows for heating is 18°, but at night I would like to keep my apartment at 16°. I'm thinking I can trick the AC into thinking the target temperature is already reached by adding 2° to the thermistor.

Would this work?

**Comment by JanM321** [comment]
Yes I think that will work fine but I haven't tried this for a longer period of time. You can do this with a HA template sensor.
